packages/prop-types/README.md

```diff
@@ -19,3 +19,39 @@ Box.propTypes = {
   ...propTypes.color,
 }
 ```
+
+## Available Type Categories
+
+* `space`
+* `color`
+* `layout`
+* `typography`
+* `flexbox`
+* `border`
+* `background`
+* `position`
+* `grid`
+
+See props of each category in [the reference table](https://styled-system.com/table).
+
+## Custom Props
+
+```js
+import styled from 'styled-components'
+import { space, style } from 'styled-system'
+import propTypes, { propType } from '@styled-system/prop-types'
+
+const gap = style({
+	prop: 'gap',
+	cssProperty: 'gridGap',
+	key: 'space',
+	scale: DEFAULT_SCALE,
+});
```

Comment:
Thanks! This looks good, but I would like to promote the new `system` API instead of the v4 `styled` API for examples like this. E.g.

```js
const gridGap = system({
  gap: {
    property: 'gridGap',
    scale: 'space',
    defaultScale: DEFAULT_SCALE,
  }
})
const Stack = styled('div')(
  compose(space, gridGap)
)
```